server/views/pages/overview/partials/timelineTab/timelineTabContents.test.ts

```diff
@@ -0,0 +1,98 @@
+import nunjucks from 'nunjucks'
```

Comment:
`timelineTabContents.njk` didn't have a test previously, even though there was a lot of branching logic! 😬 

Having refactored `timelineTabContents.njk` these tests assert that:
* the correct components are included based on the array of events
* the "empty timeline" content is displayed given an empty timeline
* the "timeline unavailable" message is displayed given some problem retrieving the timeline from the API